src/clients/box_ui/lib/src/widgets/autocomplete_listItem.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:box_ui/box_ui.dart';
+import 'package:flutter/material.dart';
+
+class AutoCompleteListItem extends StatelessWidget {
+  final String state, city;
+  final Function onTap;
+
+  const AutoCompleteListItem(
+      {Key? key, required this.state, required this.city, required this.onTap})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      mainAxisSize: MainAxisSize.min,
+      children: [
+        ListTile(
+          contentPadding: EdgeInsets.zero,
+          leading: Icon(
+            Icons.near_me,
+            color: kcMediumGreyColor,
+          ),
+          onTap: () => onTap(),
+          title: BoxText.subheading(city),
+          subtitle: BoxText.caption(state),
+        ),
+        Container(margin: EdgeInsets.only(left: 50.0), child: Divider())
```

Comment:
Divider has a `indent` and `endIndent` param.
```dart
Container(margin: EdgeInsets.only(left: 50.0), child: Divider())
````
simplifies to
```dart
const Divider(indent: 50),
```

Reply:
Thanks a lot Victor. Will update the changes.